CorrelatorSharp/packages.config

```diff
@@ -1,4 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm not too familiar with nuspec/nuget configurations.  I may be mistaken but I believed when there is no packages.config it should check the nuspec dependencies and pull them down if required? I performed a clean checkout and build locally and it seemed to be okay but maybe there was some unexpected caching somewhere... I'll add packages.config back in and make immutable collections a developmentDependency.